src/main/java/org/broadinstitute/hellbender/tools/exome/acsconversion/SimpleBalancedSegmentCaller.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.hellbender.tools.exome.acsconversion;
+
+
+import org.broadinstitute.hellbender.tools.exome.ACNVModeledSegment;
+
+public class SimpleBalancedSegmentCaller implements BalancedSegmentCaller{
+
+    public SimpleBalancedSegmentCaller() {}
+
+    /**
+     * Extremely simple method to classify an ACNVModeledSegment as balanced (i.e. MAF is exactly 0.5) or not.
+     *
+     * @param segment Never {@code null}
+     * @return whether this segment should be considered balanced
+     */
+    public boolean isSegmentBalanced(final ACNVModeledSegment segment) {
+        return segment.getMinorAlleleFractionPosteriorSummary().upper() > 0.49;
```

Comment:
I think we may want to bring some measure of the width (upper - center, for example) into play here, but let's see how this works out for CGA first.